channels/us.m3u

```diff
@@ -530,13 +530,13 @@ https://bloomberg-bloombergtv-1-es.samsung.wurl.com/manifest/playlist.m3u8
 https://liveprodusphoenixeast.akamaized.net/USPhx-HD/Channel-TX-USPhx-AWS-virginia-1/Source-USPhx-16k-1-s6lk2-BP-07-02-81ykIWnsMsg_live.m3u8
 #EXTINF:-1 tvg-id="BloombergUSDesktop.us" tvg-name="Bloomberg US Desktop" tvg-country="US" tvg-language="English" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg US Desktop
```

Comment:
I don't think it's really called US Desktop

Reply:
I have seen that last week, I think they were trying to say the website version, will go take a look.

Reply:
That version does look different, seems like a desktop app

Reply:
Maybe we should name it Bloomberg Anywhere

Reply:
Link above the US Desktop is the same.

Reply:
Line 530 and 532 are not the same, not an exact match on my local branch

Reply:
Are you saying it might be Bloomberg TV+ I checked there are many of those and they're broadcasting the same content. I will rename it to Bloomberg TV+